_posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title:  "A Report about Presto Conference Tokyo 2020 Online"
+author: Toru Takahashi, Treasure Data
+excerpt_separator: <!--more-->
+---
+
+On Nov 11th, 2020, Japan Presto Community has held the 2nd Presto Conference 
+with welcoming Martin Traverso and Brian.
+The conference was hosted at Youtube Live.
+This article is the summary of the conference aiming to share their great talks. 
+
+<!--more-->
+
+# Presto Community Updates
+
+First of all, Martin introduce recent Presto updates in these days. 
+It covers rencent changes and enhancements achieved by the community activities.
+Attendees also learned several new functions that will be available soon.
+
+- Update / Merge (https://github.com/prestosql/presto/issues/3325)
+- Materialized Views (https://github.com/prestosql/presto/pull/3283)
+- Dynamically resolved functions
+- Optimized Parquet reader
+
+In addition, at Q&A, he suggests new developers who want to contribute to PrestoSQL 
+to check "good first issue" tag on Github. The tag is a good first step for a new joiner to contribute. 
+Ref. [link](https://github.com/prestosql/presto/labels/good%20first%20issue)
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/NxDBBEA67Ws?start=189" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Use the youtube tag as used in other posts

Reply:
We can use
```
{% youtube NxDBBEA67Ws %}
```